core/domain/exp_domain_test.py

```diff
@@ -590,7 +590,7 @@ def test_content_ids_to_audio_translations_validation(self):
         hints_list.append({
             'hint_content': {
                 'content_id': 'hint_1',
-                'html': {}
+                'html': ''
```

Comment:
I'm looking into this one as I'm expecting to have a different validation error log over here (not sure!). I'll update after looking into this in detail.

Reply:
Sure. Thanks @DubeySandeep